package.json

```diff
@@ -47,10 +48,13 @@
     "changelog": "standard-version",
     "inspect": "node --inspect dist/index.js",
     "release": "yarn reset && yarn test && yarn changelog",
+    "clean": "trash dist",
     "reset": "git clean -dfx && git reset --hard && yarn",
     "ci": "yarn test",
     "validate:dependencies": "yarn audit --groups dependencies",
-    "license-validate": "node-license-validator -p -d --allow-licenses MIT MIT/X11 BSD BSD-3-Clause BSD-2-Clause ISC Apache Apache-2.0 Unlicense --allow-packages cycle",
+    "validate": "yarn audit --level moderate --groups dependencies --recursive",
+    "validate-license": "license-checker --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense",
+    "validate-summary": "license-checker --summary --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense",
```

Comment:
`validate` should both run audit and license validation. I'd suggest doing something like the following:
```typescript
    "validate": "yarn validate-dependencies && yarn validate-licenses",
    "validate-dependencies": "yarn audit --groups dependencies",
    "validate-licenses": "license-checker --production --onlyAllow MIT;MIT/X11;BSD;BSD-3-Clause;BSD-2-Clause;ISC;Apache;Apache-2.0;Unlicense",
```